test/unit-tests.c

```diff
@@ -806,12 +806,14 @@ test_dma_addr_to_sg(void **state __attribute__((unused)))
     assert_true(sg.mappable);
 
     r->prot = PROT_NONE;
-    assert_int_equal(-EACCES,
+    assert_int_equal(-1,
```

Comment:
We should also have a unit test where we call `dma_addr_to_sg` which fails because a region is not found (must return -1  but `errno` must be 0).